composer.lock

```diff
@@ -0,0 +1,2719 @@
+{
```

Comment:
Hey, please don't commit the composer.lock

Reply:
Why shouldn`t we commit the composer.lock? This ensures, that every developer has the same dependency versions. Its also recommended from the composer docu. [Composer Doku](https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control)